wizard/WizardHome.qml

```diff
@@ -56,10 +57,46 @@ Rectangle {
             Layout.alignment: Qt.AlignHCenter
             spacing: 0
 
-            WizardHeader {
-                Layout.bottomMargin: 20
-                title: qsTr("Welcome to Monero") + translationManager.emptyString
-                subtitle: ""
+            RowLayout {
+                Layout.fillWidth: true
+                spacing: 10
+
+                WizardHeader {
+                    Layout.bottomMargin: 7
+                    title: qsTr("Welcome to Monero") + translationManager.emptyString
+                    subtitle: ""
+                }
+
+                Rectangle {
+                    Layout.fillWidth: true
+                    height: 10
+                    color: "transparent"
+                }
+
+                MoneroComponents.Label {
+                    Layout.bottomMargin: 13
+                    fontSize: 26
+                    text: FontAwesome.language
+                    fontFamily: FontAwesome.fontFamilySolid
+                    fontColor: MoneroComponents.Style.defaultFontColor
+                    styleName: "Solid"
+                    opacity: 0.85
+                }
```

Comment:
Shouldn't we change icon's opacity on mouse hover?

Reply:
And also clickable

Reply:
My initial idea was the same. I tried to put both Icon and language text inside a rectangle and then set a `MouseArea` with `anchors.fill: parent`, but I couldn't get rid of the error "_detected anchors inside item managed by Layout, use Layout.alignment instead_" neither make the MouseArea fill the Layout's width and height (it should overlap/occupy the same space of icon and language text items). So, instead of setting two MouseAreas (one for each item), I decided to set just one for the language text.

Reply:
Add another RowLayout containing both the icon and the text.